lib/debuglet.js

```diff
@@ -31,22 +34,14 @@ var NODE_VERSION_MESSAGE = 'V8 Debug API not available – make sure you are ' +
     'running Node.js >=0.12. Cloud Debugger agent will not be able to take ' +
     'snapshots.';
 
-module.exports = {
-  start: function(config, logger, exports) {
-    var agent = new Debuglet(config, logger);
-    agent.start(function(err) {
-      // For e2e testing.
-      if (!err) {
-        exports.debug.api_ = agent.debugletApi_;
-      }
-    });
-
-    exports.private_ = agent;
-  }
-};
+module.exports = Debuglet;
 
 /**
  * @param {Object} config The option parameters for the Debuglet.
+ * @event 'error'
+ * @event 'started'
+ * @event 'registered'
```

Comment:
Does registered have the connotation of "successfully initialized"? Could we document what an event of this type means.